OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameCashCounterLogic.cs

```diff
@@ -42,7 +44,7 @@ public IngameCashCounterLogic(Widget widget, World world)
 			displayLabel = cashLabel.F(displayResources);
 
 			cash.GetText = () => displayLabel;
-			cash.GetTooltipText = () => $"Silo Usage: {playerResources.Resources}/{playerResources.ResourceCapacity}";
+			cash.GetTooltipText = () => modData.Translation.GetString(SiloUsage) + $": {playerResources.Resources}/{playerResources.ResourceCapacity}";
```

Comment:
Shouldn't the "amount / capacity" be part of the translation string?

Also, pull the GetString outside, using a `CachedTransform` after fixing ^.

Reply:
`CachedTransform` does not support two arguments.